libs/cms/src/lib/cms.contentful.service.ts

```diff
@@ -1,3 +1,5 @@
+// TRIGGER LINE FOR FEATURE DEPLOY WORKAROUND - REMOVE BEFORE MERGE
+
```

Comment:
Is this here to make sure the contentful service is deployed and launched? Why isn't that happening automatically?

Reply:
no this ... regretfully ... is how triggering a #feature-deploy build works for services such as user-notification ... it´s a hack workaround which infra will address